staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/width.go

```diff
@@ -83,9 +89,17 @@ func (e WorkEstimatorFunc) EstimateWork(r *http.Request, flowSchemaName, priorit
 	return e(r, flowSchemaName, priorityLevelName)
 }
 
+// WorkEstimator is a wrapper around WorkEstimatorFunc that also contains a function for
+// calculating ratio of estimated and maximum number of seats associated with a request.
```

Comment:
s/estimated and maximum/maximum (of initial and final) estimate divided by the configured bound on the/

Reply:
Changed.